apps/api/src/server/swagger-output.json

```diff
@@ -7876,6 +7944,11 @@
 					"description": "Published status",
 					"example": "published"
 				},
+				"isDeleted": {
```

Comment:
Where is this being used? Can't see it anywhere

Reply:
I added that in to the swagger.js in the definition S51AdviceDetailsWithCaseId, as with the addition of that new field, the certain existing APIs that return S51AdviceDetailsWithCaseId now also return that field.